catt/cli.py

```diff
@@ -103,28 +107,111 @@ def write_config(settings):
         raise CattCliError("No device specified.")
 
 
+def hunt_subtitle(video):
+    """"Searches for subtitles in the current folder"""
+    video_path = Path(video)
+    video_path_stem_lower = video_path.stem.lower()
+    for entry in video_path.parent.iterdir():
+        entry_path = Path(entry)
+        if entry_path.is_dir():
+            continue
+        if entry_path.stem.lower().startswith(video_path_stem_lower) and \
+                entry_path.suffix.lower() in [".vtt", ".srt"]:
+            return str(entry_path.resolve())
+    return None
+
+
+def convert_srt_to_webvtt_helper(content):
+    content = re.sub(r'([\d]+)\,([\d]+)', r'\1.\2', content)
```

Comment:
This seems a bit too general, it will catch things like internationalized numbers in the subtitles and mistakenly convert them. Can we make it more specific?